core/lib/zksync_core/src/api_server/web3/tests/mod.rs

```diff
@@ -283,7 +342,29 @@ async fn store_events(
             indexed_topics: vec![H256::repeat_byte(42), H256::repeat_byte(111)],
             value: (start_idx + 3).to_le_bytes().to_vec(),
         },
+        // ETH Transfer event with matching both address and topic
+        VmEvent {
```

Comment:
Bikeshedding: IMO, it would be better to have a dedicated setup for ETH transfer events; i.e., a method like `store_eth_transfer_events()`. To avoid copy-paste, common code between `store_events` and `store_eth_transfer_events` can be extracted to a method that accepts `&[VmEvent]` as an argument. Or, you can modify `store_events` to accept such an argument and create methods to generate "default" and ETH transfer events.